src/app/components/pages/address-detail/address-detail.component.ts

```diff
@@ -289,12 +289,20 @@ export class AddressDetailComponent implements OnInit, OnDestroy {
         });
       }
 
+      // If no transactions were found for the address.
+      if (this.totalTransactionsCount < 1) {
+        // Show that there are no transactions.
+        this.longErrorMsg = 'addressDetail.withoutTransactions';
+        // Needed for showing the previous msg.
+        this.alltransactions = undefined;
+      }
+
       this.dataLoaded = true;
     }, error => {
       if (error.status >= 400 && error.status < 500) {
         // The address was not found.
         this.loadingMsg = 'general.noData';
-        this.longErrorMsg = 'addressDetail.withoutTransactions';
+        this.longErrorMsg = 'addressDetail.invalidAddress';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/skycoin/skycoin-explorer/pullRequest?prid=7163626)
```suggestion
        this.longErrorMsg = "addressDetail.invalidAddress";
```